modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1263,8 +1272,8 @@ The query engine first scans the input values shown by the operator `ValueScan`
 .Query
 [source,n1ql]
 ----
-EXPLAIN INSERT INTO `travel-sample` (key UUID(), value _airport)
-    SELECT _airport FROM `travel-sample` _airport
+EXPLAIN INSERT INTO `travel-sample`.inventory.airport (key UUID(), value airport)
+    SELECT airport FROM `travel-sample`
       WHERE type = "airport" AND airportname = "Heathrow";
```

Comment:
I think the source should be the airport collection also. So effectively this just creates a copy of the Heathrow record. Unfortunately this change means you'll have to update the results of this query again, sorry about that.
```suggestion
EXPLAIN INSERT INTO `travel-sample`.inventory.airport (key UUID(), value airport)
    SELECT airport FROM `travel-sample`.inventory.airport
      WHERE airportname = "Heathrow";
```